src/Abstractions/NexusMods.Abstractions.NexusModsLibrary/Models/Collection.cs

```diff
@@ -0,0 +1,54 @@
+using NexusMods.Abstractions.MnemonicDB.Attributes;
+using NexusMods.Abstractions.NexusModsLibrary.Attributes;
+using NexusMods.MnemonicDB.Abstractions.Attributes;
+using NexusMods.MnemonicDB.Abstractions.Models;
+
+namespace NexusMods.Abstractions.NexusModsLibrary.Models;
+
+/// <summary>
+/// Metadata about a collection on Nexus Mods.
+/// </summary>
+public partial class Collection : IModelDefinition
+{
+    private const string Namespace = "NexusMods.Library.NexusModsCollectionMetadata";
+    
+    /// <summary>
+    /// The collection slug.
+    /// </summary>
+    public static readonly CollectionsSlugAttribute Slug = new(Namespace, nameof(Slug)) { IsIndexed = true };
+    
+    /// <summary>
+    /// The name of the collection.
+    /// </summary>
+    public static readonly StringAttribute Name = new(Namespace, nameof(Name));
+    
+    /// <summary>
+    /// The short description of the collection
+    /// </summary>
+    public static readonly StringAttribute Summary = new(Namespace, nameof(Summary));
+    
+    /// <summary>
+    /// The Curating user of the collection.
+    /// </summary>
+    public static readonly ReferenceAttribute<User> User = new(Namespace, nameof(User));
```

Comment:
Should be named author or curator.

Reply:
It's called `User` in GraphQL

Reply:
Right, but we don't have to keep that name. `Author` or `Curator` would be more descriptive. As seen in the usage:

```csharp
    public string AuthorName => _collection.User.Name;
```

Reply:
Swapped it over to author